README.md

```diff
@@ -32,15 +34,21 @@ Expand **Qt > Qt 6.2.1 > Additional Libraries** and select **Qt multimedia**
 
 This library is required to output sound out of QT
 
-
 ### Building Error FAQ
 
 must have VS2019 installed for compiler
 
 deselect shadow build
 
-
-Contributors: 
+Contributors:
 Max
 Kyu Bum Kim (kb.kim@mail.utoronto.ca)
 Eduardo Ortenblad (edu.ortenblad@mail.utoronto.ca)
+
+Acknowledgements:
+
+This code uses STK
+
+This code uses QCustomPlot which is built on the QT
```

Comment:
This code uses QCustomPlot which is built on QT

Reply:
changed